OrbitService/LinuxTracingHandler.cpp

```diff
@@ -144,9 +144,9 @@ void LinuxTracingHandler::OnTracepointEvent(orbit_grpc_protos::TracepointEvent t
 }
 
 void LinuxTracingHandler::OnModulesUpdate(
-    orbit_grpc_protos::ModuleUpdateEvent modules_update_event) {
+    orbit_grpc_protos::ModuleUpdateEvent module_update_event) {
```

Comment:
Can you rename it also in the header file and in `TracerListener`?

Reply:
done.